components/Members/Members.js

```diff
@@ -0,0 +1,130 @@
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { useState, useEffect } from 'react';
+import Image from "next/image";
+
+
+async function retrieveMembers(groupName) {
+  const response = await fetch('http://35.86.78.63:8000/user/retrieve-members/' + `${groupName}`);
+  return response; // Note: Can I make this response.json()?
+}
+
+function Members(props) {
+  const [scrollItems, setScrollItems] = useState();
+  const [hasMore, setHasMore] = useState(true);
+  const [members, setMembers] = useState();
+
+  useEffect(() => {
+    retrieveMembers(props.groupName).then(
+      result => result.json()).then(
+        data => {
+          setMembers(data.members);
+          setScrollItems(data.members.slice(0, 11))
+        }
+      );
+  }, []);
+
+  const style = {
+    height: 30,
+    border: "1px solid green",
+    margin: 6,
+    padding: 8
+  };
+
+  const fetchMoreData = () => {
+    setScrollItems(scrollItems => [...scrollItems, ...members.slice(scrollItems.length - 1, members.length - 1)]);
+  };
+
+  useEffect(() => {
+    if (members?.length <= scrollItems?.length) {
```

Comment:
Why is this being set with a `useEffect`?

Reply:
`react-infinite-scroll-component` was not running `fetchMoreData` after all of the `items` in `members` state were displayed so it showed a "Load..." message instead of a finished message. This is a minor error with the dependency.